src/remote/inet.cpp

```diff
@@ -1097,7 +1096,7 @@ static rem_port* listener_socket(rem_port* port, USHORT flag, const addrinfo* pa
 	//			e.g. while it's listening. This is surely not what we want.
 	//			We set this options for any kind of listener, including standalone Classic.
 
-	int optval = TRUE;
+	constexpr int optval = TRUE;
```

Comment:
Are you sure that getting address of `constexpr` is a defined behavior? Cast below perhaps raise warning about qualifier being dropped.

Reply:
And that is, BTW, because the parameter to setopt is defined as a pointer to const.

Reply:
> Are you sure that getting address of constexpr is a defined behavior?

Yes, it's basically the same as a const variable, but its value should be calculated at compile time.